src/features/game/expansion/components/UpcomingExpansion.tsx

```diff
@@ -73,13 +72,17 @@ export const UpcomingExpansion: React.FC<Props> = ({ gameState }) => {
         onHide={() => setShowBumpkinModal(false)}
         centered
       >
-        <Panel bumpkinParts={gameState.bumpkin?.equipped}>
-          <UpcomingExpansionModal
-            gameState={gameState}
-            onClose={() => setShowBumpkinModal(false)}
-            onExpand={onExpand}
-          />
-        </Panel>
+        <CloseButtonPanel
+          bumpkinParts={gameState.bumpkin?.equipped}
+          title={
+            gameState.expansionRequirements === undefined
```

Comment:
<img width="303" alt="Screen Shot 2023-01-16 at 9 17 16 am" src="https://user-images.githubusercontent.com/11745561/212570068-1d957d82-e672-40bf-950e-46df9595cab5.png">

The alignment of these modals is a bit off now. The title is center aligned, but all of the content is left aligned.

Was this intended?

Reply:
I will shorten the title and move the text to the body so they are all left aligned in future PRs